core/urls.py

```diff
@@ -38,10 +38,13 @@ def get_urls(cls, message):
     @classmethod
     def get_music_urls(cls, message):
         urls = cls.get_urls(message)
+        unique_urls = []
```

Comment:
I think that can be a `set` since urls are unique

Reply:
I change but it has no effect.

Reply:
@alexshavelev44 @Ignisor I think setcomp will do the trick. Somethink like that:
```def get_music_urls(cls, message):
        urls = cls.get_urls(message)
        unique_urls = {cls.__to_music_url(url) for url in urls}
        return unique_urls
```

Reply:
@dmitrysus @Ignisor 
1) it brokes logic with yield (app always use generators instead of lists/sets)
2) it will create unnecessary classes

but maybe i'm wrong

Reply:
> I change but it has no effect.
https://stackoverflow.com/questions/2831212/python-sets-vs-lists

@alexshavelev44 effect is invisible

> but maybe i'm wrong
Nope, you right here